.cookiecutter/includes/conf/supervisord.conf

```diff
@@ -0,0 +1,20 @@
+[program:h-beat]
+command=celery -b %(ENV_H_BROKER_URL)s -A h_periodic.h_beat beat --pidfile=h-celerybeat.pid --loglevel INFO
+stdout_logfile=NONE
+stderr_logfile=NONE
+stdout_events_enabled=true
+stderr_events_enabled=true
+
+[program:checkmate-beat]
+command=celery -b %(ENV_CHECKMATE_BROKER_URL)s -A h_periodic.checkmate_beat beat --pidfile=checkmate-celerybeat.pid --loglevel INFO
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+
+[program:lms-beat]
+command=celery -b %(ENV_LMS_BROKER_URL)s -A h_periodic.lms_beat beat --pidfile=lms-celerybeat.pid --loglevel INFO
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
```

Comment:
As you can see, we're forced to put the same information in `.cookiecutter/includes/conf/supervisord.conf` and `.cookiecutter/includes/conf/supervisord-dev.conf`. Also, the `stdout_logfile`, `stderr_logfile`, `stdout_events_enables`, `stderr_events_enabled`, `stopsignal` and `stopasgroup` should all be inserted by the cookiecutter by default for each program. We can allow each project to override these settings, but the project's shouldn't be forced to repeat the default settings.

We'll need an enhancement to the `supervisord[-dev].conf` templates in the cookiecutter in order to be able to do this properly.